src/mainboard/ast/ast25x0/Cargo.lock

```diff
@@ -1,323 +0,0 @@
-# This file is automatically @generated by Cargo.
```

Comment:
I think Cargo.lock files in src/mainboard/** should be checked in

Reply:
I thought we talked about this last Friday and said the opposite, but clearly I misunderstood.  Will fix shortly.